app/producers/report_validation.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# A Kafka producer client for upload-compliance
+class ReportValidation < ApplicationProducer
+  TOPIC = Settings.kafka.topics.upload_compliance
+  DATE_FORMAT = :rfc3339
```

Comment:
The `rfc3339` was only set to fix PayloadTracker and might need to be removed from here to return to the default ISO 8601.

See https://github.com/RedHatInsights/compliance-backend/commit/43082eea6b82e08e39ed408bfac4726caec2bbb1